firmware/Makefile

```diff
@@ -59,6 +59,7 @@ CFLAGS += -Wno-sequence-point
 CFLAGS += -Iprotob -DPB_FIELD_16BIT=1
 CFLAGS += -DQR_MAX_VERSION=0
 CFLAGS += -DDEBUG_LINK=0
+CFLAGS += -DDEBUG_RNG=0
```

Comment:
yep, understood. just a matter of should i keep it consistent with how DEBUG_LINK is configured?